gadgets/audit_seccomp/go/program.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2025 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"slices"
+
+	api "github.com/inspektor-gadget/inspektor-gadget/wasmapi/go"
+
+	ocispec "github.com/opencontainers/runtime-spec/specs-go"
+
+	seccomp "main/seccomp"
+)
+
+const (
+	// config.json is typically less than 100 KiB.
+	// 1 MiB should be enough.
+	configJsonMaxSize = 1 * 1024 * 1024
+)
+
+func updateConfig(configText string) string {
+	config := &ocispec.Spec{}
+	err := json.Unmarshal([]byte(configText), config)
+	if err != nil {
+		api.Errorf("unmarshaling config.json: %s", err)
+		return ""
+	}
+
+	if config.Linux == nil {
+		api.Errorf("Linux config is nil\n")
+		return ""
+	}
+	if config.Linux.Seccomp == nil {
```

Comment:
Why create one if it's nil? Isn't the idea just to add it to containers that have seccomp, to debug them easily?

Reply:
You're right, I got things mixed up between the audit_seccomp gadget and the advise_seccomp gadget.